### PR TITLE
Compat data for missing `file_handlers` manifest member

### PR DESCRIPTION
**Pull request by captainbrosset**
#### Summary

`file_handlers` is a web app manifest member that can be used to register an install PWA as a handler for files at the OS level.

#### Test results and supporting details

- Docs
  - https://developer.chrome.com/articles/file-handling/
  - https://learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/handle-files
- Feature tracking
  -  https://chromestatus.com/feature/5721776357113856

### CODE REVIEW

File: html/manifest/file_handlers.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "manifest": {
+      "file_handlers": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/Manifest/file_handlers",
+          "spec_url": "https://wicg.github.io/manifest-incubations/index.html#file_handlers-member",
+          "support": {
+            "chrome": {
+              "version_added": "102"
+            },
+            "chrome_android": "mirror",
```
Comment (tomayac) [inline html/manifest/file_handlers.json:12]:
```suggestion
            "chrome_android": false,
```

It’s desktop-only so far.

**Review by tomayac — APPROVED** [review]
LGTM with the Chrome Android correction.

File: html/manifest/file_handlers.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      "file_handlers": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/Manifest/file_handlers",
+          "spec_url": "https://wicg.github.io/manifest-incubations/index.html#file_handlers-member",
+          "support": {
+            "chrome": {
+              "version_added": "102"
+            },
+            "chrome_android": {
+               "version_added": false
```
Comment (Elchi3) [inline html/manifest/file_handlers.json:13]: Sorry, more CI complaints. 

```suggestion
              "version_added": false
```

File: html/manifest/file_handlers.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": {
+              "version_added": false
+            }
```
Comment (Elchi3) [inline html/manifest/file_handlers.json:33]: Apparently CI wants us to use "mirror" for webview_android given it matches chrome_android now.

```suggestion
            "webview_android": "mirror",
```

**Review by Elchi3 — CHANGES_REQUESTED** [review]

**Review by Elchi3 — APPROVED** [review]
Thanks @captainbrosset! Usually it is fine but yeah, you have more power locally when running tests etc.

**Pull request merged**